types/chat.ts

```diff
@@ -109,15 +110,19 @@ export interface InsertToCursorPositionParams {
     totalCodeBlocks?: number
 }
 
-export interface LinkClickParams {
+export interface InfoLinkClickParams {
     tabId: string
-    messageId: string
     link: string
     mouseEvent?: MouseEvent
```

Comment:
I suggest we remove mouseevent completely at this point while we are making changes